esaview

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
 eval `/proj/sot/ska/bin/flt_envs -shell sh`
-exec /proj/sot/ska/share/taco/esaview.py "$@"
+exec /proj/sot/ska/share/acis_taco/esaview.py "$@"
```

Comment:
I thought the Makefile installed esaview.py to $SKA/share, and I thought the Makefile was deleted, so not sure how this is install/path works.

Reply:
script/esaview would need to have a path to exec SKA_ARCH_OS/share/acis_taco/esaview.py